src/databases/OpenPMD/OpenPMDClasses/PMDFile.C

```diff
@@ -128,6 +129,14 @@ void PMDFile::OpenFile(char * PMDFilePath)
 
     H5Pclose(fileAccessPropListID);
 
+    // Check for error opening the file
+    if (fileId < 0)
+    {
+        debug5 << "The current file is not an openPMD file" << endl;
```

Comment:
Are debug *and* exception lines *both* needed here. Exceptions will get logged to debug files.